packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1237,6 +1238,119 @@ export class Profiles extends ProfilesCache {
         }
     }
 
+    public async basicAuthClearSecureArray(profileName?: string): Promise<void> {
+        const profAttrs = await this.getProfileFromConfig(profileName);
+        const configApi = (await this.getProfileInfo()).getTeamConfig();
+        configApi.set(`${profAttrs.profLoc.jsonLoc}.secure`, ["tokenValue"]);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.user`);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.password`);
```

Comment:
This works when user & password are stored on the service profile directly, but it doesn't work if they are stored in a base profile.

Instead of hardcoding the JSON location where the secure properties are stored, we may want to use the `configApi.api.secure.securePropsForProfile` API: https://docs.zowe.org/stable/typedoc/classes/_zowe_imperative.configsecure.html#securepropsforprofile